packages/SwingSet/test/test-netstring.js

```diff
@@ -0,0 +1,60 @@
+import '@agoric/install-ses'; // adds 'harden' to global
+import test from 'ava';
+import { encode, decode } from '../src/netstring';
+
+test('encode', t => {
+  function eq(input, expected) {
+    const encoded = encode(Buffer.from(input));
+    const expBuf = Buffer.from(expected);
+    if (encoded.compare(expBuf) !== 0) {
+      console.log(`got : ${encoded}`);
+      console.log(`want: ${expBuf}`);
+    }
+    t.deepEqual(encoded, expBuf);
+  }
+
+  eq('', '0:,');
+  eq('a', '1:a,');
+  eq('abc', '3:abc,');
+  const umlaut = 'ümlaut';
```

Comment:
Could also test `👨‍👨‍👧‍👧` or something equally elaborate to not just test very high code points but also combining characters.

Reply:
added this, thanks